src/layouts/AllServices.tsx

```diff
@@ -0,0 +1,542 @@
+                  </CardTitle>
+                  <CardBody>
+                    <TextContent>
+                      <Text component={TextVariants.p}>Ensure that the right users have the appropriate access to technology resources.</Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="#">
+                          Authentication policy
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/settings/my-user-access">
+                          My User Access
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/openshift/create">
+                          User Access
+                        </Text>
+                      </Text>
+                    </TextContent>
+                  </CardBody>
+                </Card>
+                <Card isPlain>
+                  <CardTitle>
+                    <InfrastructureIcon /> Infrastructure
+                  </CardTitle>
+                  <CardBody>
+                    <TextContent>
+                      <Text component={TextVariants.p}>Manage your infrastructure across the hybrid cloud.</Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/openshift">
+                          Clusters
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/openshift/overview">
+                          OpenShift
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/openshift/releases">
+                          Releases
+                        </Text>
+                      </Text>
+                    </TextContent>
+                  </CardBody>
+                </Card>
+                <Card isPlain>
+                  <CardTitle>
+                    <BellIcon /> Integration and Notifications
+                  </CardTitle>
+                  <CardBody>
+                    <TextContent>
+                      <Text component={TextVariants.p}>Alerts users to events, using email and integrations such as Webhooks.</Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="#">
+                          Integration (Sources)
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="#">
```

Comment:
I think it's fine to navigate user to a `user-preferences/notifications/application-services`

Reply:
@karelhala "Integration and Notifications" is still WIP. @KendraMar

Reply:
@epwinchell Yup...I'll be talking with Kat on Friday, try to finalize what's in that category.